crates/wasmtime/src/config.rs

```diff
@@ -682,6 +682,47 @@ impl Config {
         self
     }
 
+    /// Configures whether the WebAssembly Relaxed SIMD proposal will be
+    /// enabled for compilation.
+    ///
+    /// The [WebAssembly Relaxed SIMD proposal][proposal] is not, at the time of
+    /// this writing, at stage 4. Additionally note that the relaxed SIMD
+    /// proposal's instructions are allowed to have different semantics on
+    /// different architectures depending on the inputs. By default the fastest
+    /// this difference in semantics is enabled in Wasmtime, but this behavior
```

Comment:
I'm not sure what this sentence was supposed to say. Maybe delete "the fastest", or replace "this difference in" with something else?